13/umbraco-deploy/deployment-workflow/import-export.md

```diff
@@ -344,22 +346,165 @@ public class NestedContentPropertyTypeMigrator : PropertyTypeMigratorBase
+{
+    protected override void ApplicationInitialized(UmbracoApplicationBase umbracoApplication, ApplicationContext applicationContext)
+    {
+        // Set a default value connector that doesn't use object type prefixes
+        DefaultValueConnector.SetDefault();
+
+        // Run export after Deploy has started
+        DeployComponent.Started += (sender, e) => DeployStarted();
+    }
+
+    protected void DeployStarted()
+    {
+        var udis = new[]
+        {
+            // Export all content
+            Constants.UdiEntityType.Document,
+            Constants.UdiEntityType.DocumentBlueprint,
+            Constants.UdiEntityType.Media,
+            // Export all forms data
+            Constants.UdiEntityType.FormsForm,
+            Constants.UdiEntityType.FormsDataSource,
+            Constants.UdiEntityType.FormsPreValue
+        }.Select(Udi.Create);
+
+        var dependencyEntityTypes = new[]
+        {
+            // Include all related schema
+            Constants.UdiEntityType.DataType,
+            Constants.UdiEntityType.DataTypeContainer,
+            Constants.UdiEntityType.DocumentType,
+            Constants.UdiEntityType.DocumentTypeContainer,
+            Constants.UdiEntityType.MediaType,
+            Constants.UdiEntityType.MediaTypeContainer,
+            Constants.UdiEntityType.MemberType,
+            Constants.UdiEntityType.MemberGroup,
+            Constants.UdiEntityType.Macro,
+            Constants.UdiEntityType.DictionaryItem,
+            Constants.UdiEntityType.Template,
+            Constants.UdiEntityType.Language,
+            // Include all related files
+            Constants.UdiEntityType.MediaFile,
+            Constants.UdiEntityType.MacroScript,
+            Constants.UdiEntityType.PartialView,
+            Constants.UdiEntityType.PartialViewMacro,
+            Constants.UdiEntityType.Script,
+            Constants.UdiEntityType.Stylesheet,
+            Constants.UdiEntityType.UserControl,
+            Constants.UdiEntityType.TemplateFile,
+            Constants.UdiEntityType.Xslt
+        };
+
+        // Create export
+        var zipArchiveFilePath = HostingEnvironment.MapPath("~/data/" + "export-" + Guid.NewGuid() + ".zip");
+        ArtifactExportService.ExportArtifacts(udis, Constants.DeploySelector.ThisAndDescendants, zipArchiveFilePath, dependencyEntityTypes);
+    }
+}
+```
+</details>
+
+#### Importing Umbraco 7 content and schema
+
+To import this archive into a newer Umbraco project, you need to install `UmbracoDeploy.Contrib` 4.3 (for Umbraco 8) or `Umbraco.Deploy.Contrib` 10.2, 12.1 or 13.1 (or later) and configure the legacy artifact type resolver and migrators. Artifact type resolvers allow resolving changes in the type that's stored in the `__type` JSON property of the artifact, in case it moved to a different assembly or namespace (or got renamed) in a newer version. The legacy migrators handle the following changes:
```

Comment:
```suggestion
To import this archive into a newer Umbraco project, you need to either install:
* `UmbracoDeploy.Contrib` 4.3 for Umbraco 8
*`Umbraco.Deploy.Contrib` for Umbraco 10.2, 12.1, 13.1 or later 
You then need to configure the legacy artifact type resolver and migratory. 

Artifact type resolvers allow resolving changes in the type that's stored in the `__type` JSON property of the artifact, in case it moved to a different assembly or namespace (or got renamed) in a newer version. The legacy migrators handle the following changes:
```

Reply:
```suggestion
To import this archive into a newer Umbraco project, you need to install either of these packages:
* `UmbracoDeploy.Contrib` 4.3 for Umbraco 8
* `Umbraco.Deploy.Contrib` for Umbraco 10.2, 12.1, 13.1 or later 
Then you need to configure the legacy artifact type resolver and migratory.
```